service/cluster/xtra_db_cluster.go

```diff
@@ -106,17 +106,21 @@ func (s *XtraDBClusterService) CreateXtraDBCluster(ctx context.Context, req *con
 		Name: req.Name,
 		Size: req.Params.ClusterSize,
 	}
-	params.PXC = &k8sclient.PXC{
-		ComputeResources: &k8sclient.ComputeResources{
-			CPUM:        req.Params.Pxc.ComputeResources.CpuM,
-			MemoryBytes: req.Params.Pxc.ComputeResources.MemoryBytes,
-		},
+	if req.Params.Pxc.ComputeResources.CpuM > 0 && req.Params.Pxc.ComputeResources.MemoryBytes > 0 {
```

Comment:
we should pass one of the values if it's provided

Reply:
Fixed.